build.py

```diff
@@ -755,6 +761,17 @@ def armnn_tflite_cmake_args():
     ]
 
 
+def fastertransformer_cmake_args():
+    print("Warning: FasterTransformer backend is not officially supported.")
+    cargs = [
+        cmake_backend_arg('fastertransformer', 'CMAKE_EXPORT_COMPILE_COMMANDS',
+                          None, 1)
+    ]
+    cargs.append(
```

Comment:
It might be cleaner to initialize with this arg rather than append it after. Example [here](https://github.com/triton-inference-server/server/blob/0e677f6cc6d2ffe6fa4f21ce72b14da698436726/build.py#L584-L589).